### PR TITLE
Update databases.md

### PR DESCRIPTION
**Pull request by ankulbhardwaj**
Update SQL data warehouse to Azure Synapse Analytics with the new URL.

### CODE REVIEW

**Comment by PRMerger20** [comment]
@ankulbhardwaj : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/aws-professional/databases.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Costs for AWS RDS are determined by the amount of hardware resources that your i
 
 - [Data Factory](https://azure.microsoft.com/documentation/services/data-factory): provides data orchestration and data pipeline functionality.
 
-- [SQL Data Warehouse](https://azure.microsoft.com/documentation/services/sql-data-warehouse): large-scale relational data storage.
+- [Azure Synapse Analytics](https://azure.microsoft.com/documentation/services/synapse-analytics): large-scale datawarehouse and big data system.
```
Comment (splitfinity) [inline docs/aws-professional/databases.md:43]: I would phrase this as: Enterprise scale data warehouse and analytics platform.

**Review by splitfinity — CHANGES_REQUESTED** [review]
I think this section still requires a bit of work. Still makes reference to Cortana Intelligence Suite, which is  no longer relevant.

**Comment by PRMerger14** [comment]
@EdPrice-MSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/aws-professional/databases.md (hunk 1 above)
Comment (EdPrice-MSFT) [reply docs/aws-professional/databases.md]: Taking the change.

**Review by EdPrice-MSFT — APPROVED** [review]
Good udpate!

**Comment by EdPrice-MSFT** [comment]
#Sign-Off

**Comment by PRMerger10** [comment]
@EdPrice-MSFT: I'm sorry - only the author of this article, @splitfinity, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by PRMerger10** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
Changes validated in test PR in private repo.

**Pull request merged**